### ISSUE TITLE
Undocumented input parameter in get_cansim_table_overview

### ISSUE THREAD

**Issue by dshkol** [mountainMath/cansim#19]
What is the purpose of the `column` input parameter in the function `get_cansim_table_overview`. It's not a documented parameter, and looking at the code below I'm not sure you need to have it as an input.

```r
#' Get CANSIM table overview text
#'
#' Print table overview information as messages but requires loading the entire CANSIM table in order to display overview information.
#'
#' @param cansimTableNumber the NDM table number to load
#' @param column ???
#' @param language \code{"en"} or \code{"english"} for English and \code{"fr"} or \code{"french"} for French language versions. Defaults to English.
#' @param refresh Optionally force reload of CANSIM data, default is \code{FALSE}. CANSIM data is cached for the duration of the R session only
#'
#' @return none
#'
#' @export
get_cansim_table_overview <- function(cansimTableNumber, column, language="english", refresh=FALSE){
  info <- cansim:::get_cansim_table_info(cansimTableNumber,language=language,refresh=refresh)
  refresh=FALSE
  text <- paste0(info$`Cube Title`,"\n","CANSIM Table ",cansim:::cleaned_ndm_table_number(cansimTableNumber),"\n",
                 "Start Date: ",info$`Start Reference Period`,", End Date: ",info$`End Reference Period`,", Frequency: ",info$Frequency,"\n")
  columns <- cansim:::get_cansim_column_list(cansimTableNumber,language=language,refresh=refresh)
  for (column in columns$`Dimension name`) {
    text <- paste0(text,"\n","Column ",column)
    categories <- cansim:::get_cansim_column_categories(cansimTableNumber,column,language=language,refresh=refresh)
    text <- paste0(text, " (",nrow(categories),")","\n")
    text <- paste0(text, paste(utils::head(categories$`Member Name`,10),collapse=", "))
    if (nrow(categories)>10) text <- paste0(text, ", ...")
    text <- paste0(text,"\n")
  }
  message(text)
}
```

**Comment by mountainMath** [comment]
Yeah, that's not needed. Should be taken out.

**Comment by dshkol** [comment]
Fixed in #25 and #26
